app/src/androidTest/java/com/github/se/cyrcle/model/parking/TestInstancesParking.kt

```diff
@@ -0,0 +1,46 @@
+package com.github.se.cyrcle.model.parking
+
+class TestInstancesParking {
+
+  companion object {
```

Comment:
Why make a `companion object` within a `class` instead of just declaring an `object`?

Reply:
Isn't that the only way create static instances in Kotlin ? 
I didn't want to have to create instances of the class. 
Maybe I am missing something

Reply:
Good point, completely missed the Object keyword, will fix in the next sprint when we refactor the test instances